willeswa/sendit-app
# #161583618 Admin Dashboard

What does this PR do?
Have table and navigation mockup for the admin dashboard
Description of Task to be completed?
An admin has the navigation to see registered users as well as orders in transit of all users. Admin should be able to change the current location of the order and the status of the order
How should this be manually tested?
Clone this repo into your local computer.
Navigate to the UI folder and open the admin.html file. If you open the file in a web browser, you can see the links that would be implemented at a later stage to allow for the required functionality
Any background context you want to provide?
Some functionality will be implemented during the implementation of the backend
What are the relevant pivotal tracker stories?
#161583618
Screenshots (if appropriate)
![screenshot from 2018-11-02 10-41-53](https://user-images.githubusercontent.com/38868860/47899884-09712d80-de8c-11e8-9a7c-e786c75a775e.png)
